CHANGELOG.md

```diff
@@ -1,40 +1,35 @@
-Project History
+Change log
 ===============
 
-All notable changes to this project will be documented in this file. This project adheres to [Semantic Versioning](http://semver.org/).
+### v2.1.0-0 - 2020-05-18
```

Comment:
Maybe we should change this to 3.0.0 instead of 2.1.0

Reply:
Updated in the last commit.